src/main/scala/Garage/Garage.scala

```diff
@@ -46,17 +75,17 @@ class Garage {
 
   }
 
-  def findRemainder(vehicleToBeFixed: Vehicle): Int ={
+  def findRemainder(vehicleToBeFixed: Vehicle): Int = {
     calculateFixTime(vehicleToBeFixed) % 12
   }
 
-  def fixAllVehicles(vehicleToBeFixed: List[Vehicle], currentTotalEmployeeWorkTime: Int, workableEmployees: Int, remainder: Int,whichVehicle:Int): Unit = {
-    val isThereEnoughTimeToFix=currentTotalEmployeeWorkTime-calculateFixTime(vehicleToBeFixed(whichVehicle))
-    if (isThereEnoughTimeToFix>=0) {
-       fixVehicle(vehicleToBeFixed(whichVehicle), currentTotalEmployeeWorkTime, workableEmployees, remainder)
-      val remainderSave =findRemainder(vehicleToBeFixed(whichVehicle))
+  def fixAllVehicles(vehicleToBeFixed: List[Vehicle], currentTotalEmployeeWorkTime: Int, workableEmployees: Int, remainder: Int, whichVehicle: Int): Unit = {
+    val isThereEnoughTimeToFix = currentTotalEmployeeWorkTime - calculateFixTime(vehicleToBeFixed(whichVehicle))
```

Comment:
this check could be a separate method altogether to do less things in one method